### ISSUE TITLE
ComparisonProperty is not formatted similarly to PropertyName

### ISSUE THREAD

**Issue by sid-6581** [JeremySkinner/FluentValidation#1204]
### System Details

- FluentValidation version: 8.4
- Web Framework version: ASP.NET Core 2.2

### Issue Description

```
RuleFor(p => p.AdmitDate)
	.LessThanOrEqualTo(p => p.DischDate)
	.WithMessage(@"'{PropertyName}' must be less than or equal to '{ComparisonProperty}'.");
```

This will produce a validation message that has inconsistent formatting of the property names:

`'Admit Date' must be less than or equal to 'DischDate'.`

**Comment by JeremySkinner** [comment]
Hi yes, the ComparisonProperty placeholder isn’t run through the split-camel-case routine. You’ll need to explicitly specify ‘Disch Date’ in the message.

**Comment by sid-6581** [comment]
Thanks for clarifying. If this is by design I can easily run it through Humanizer to split it up, just wasn't sure if this was intentional.

**Comment by JeremySkinner** [comment]
It’s not a feature that was completed by the looks of it (it’s not even in the documentation). I can look at getting it in for a future release.

**Comment by JeremySkinner** [comment]
Implemented in f8cb338 which will be part of 9.0
